rdr_service/dao/pediatric_data_log_dao.py

```diff
@@ -61,3 +64,17 @@ def insert(cls, data: PediatricDataLog, session: Optional[Session] = None):
         if latest_data:
             latest_data.replaced_by = data
 
+        if callback:
+            callback(session=session, existing_data=latest_data, participant_id=data.participant_id)
+
+    @classmethod
+    def _check_new_pediatric_participant(cls, session: Session, existing_data: PediatricDataLog, participant_id: int):
+        # If we're receiving our first age_range value for the participant, then they would switch to is_pediatric.
+        # So then the last_modified date should be updated.
+        if not existing_data:
+            summary: ParticipantSummary = session.query(ParticipantSummary).filter(
+                ParticipantSummary.participantId == participant_id
+            ).one_or_none()
+
+            if summary:
+                summary.lastModified = CLOCK.now()
```

Comment:
That can vary. `record_age_range` is called by the API code without passing in a session. So the decorator sets one up and then it would commit when that context manager ends.